catalog/spatial/csw/spatial-csw-transformer/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/converter/GetRecordsResponseConverter.java

```diff
@@ -102,9 +102,16 @@ public Object unmarshal(HierarchicalStreamReader reader, UnmarshallingContext co
           reader.moveDown(); // move down to the <csw:Record> tag
           String name = reader.getNodeName();
           LOGGER.debug("node name = {}", name);
-          Metacard metacard =
-              (Metacard) context.convertAnother(null, MetacardImpl.class, transformProvider);
-          metacards.add(metacard);
+
+          try {
+            Metacard metacard =
+                (Metacard) context.convertAnother(null, MetacardImpl.class, transformProvider);
+            metacards.add(metacard);
+          } catch (ConversionException e) {
+            LOGGER.warn(
+                "Failed to convert result to a metacard. Ignoring this resource because: " + e);
```

Comment:
✏️ Concatenating the exception to the String here will just call `toString()` on the exception. That generally doesn't provide much information, so we pass the exception in as a second parameter to the `warn` method and the logging framework will print the stack trace for us.
That being said, as I'm thinking about this more, I think we should maybe log the exception at DEBUG level and have the warn message just say "Set the log level to DEBUG for more information". Otherwise, this exception could spam the logs quite a bit, depending on the scenario.

Reply:
agree with this... need to avoid the potential of logging at WARN for every metacard in a response

Reply:
Would you suggest just changing the message to debug level? 
Or leaving a smaller warn message to set to debug for more info AND an additional debug log that includes the details such as the stack trace?